packages/teleport/src/Apps/AddApp/Automatically/Automatically.tsx

```diff
@@ -73,9 +73,10 @@ export default function Automatically(props: Props) {
             </Flex>
             {!cmd && (
               <Text mb="3">
-                Provide the name and URL of your application to generate a
-                script that will automatically install and configure the App
-                service on the server that can access your application.
+                Teleport can automatically setup application access. Provide
+                the name and URL of your application to generate our
+                auto-installer script. This script will install the Teleport
+                node service and provide secure access to your application.
```

Comment:
Not sure if Application Access should be uppercased as it's the product name - it looks weird though.

```suggestion
                Teleport can automatically set up application access. Provide
                the name and URL of your application to generate our
                auto-installer script. This script will install the Teleport
                app service to provide secure access to your application.
```